tasks/install.yml

```diff
@@ -1,13 +1,8 @@
 ---
-- name: Include tasks (debian/install.yml)
-  ansible.builtin.import_tasks: debian/install.yml
-  when: ansible_distro in ("debian", "ubuntu")
-  tags: [install, debian, ubuntu]
-
-- name: Include tasks (redhat/install.yml)
-  ansible.builtin.import_tasks: redhat/install.yml
-  when: ansible_distro in ("redhat", "amazon", "centos", "fedora")
-  tags: [install, redhat, centos]
+- name: "Include tasks for install in {{ _ansible_os_family }}"
```

Comment:
Including tasks requires using the apply tags syntax to recursively add the tag to included tasks otherwise if the --tags is used when calling the ansible binary, the included tasks will not run as they are not tagged.

Importing tasks on the other hand automatically adds the tags to the imported tasks but the downside is that import_tasks can not take a variable inplace of the file name.